components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java

```diff
@@ -104,11 +104,20 @@ public APIProviderImpl(String username) throws AppManagementException {
     /**
      * Delete business owner.
      * @param businessOwnerId ID of the owner.
+     * @return
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files